folly/experimental/hazptr/example/SWMRList.h

```diff
@@ -78,42 +78,45 @@ class SWMRListSet {
     }
   }
 
-  bool add(const T v) {
+  bool add(T v) {
```

Comment:
From the commit message:

> Pass `T`s by const reference, except in `add` where we're likely to want to make a copy of the `T` anyway. In that case it would be more "STL-correct" to supply two different overloads `add(T&&)` and `add(const T&)`, but this is just an example so it makes sense to keep things simple.

Changing it to `add(T&& v)` without also supplying the `add(const T&)` overload would prevent people from doing e.g.

    int i = 42;
    mylist.add(i);  // error --- "i" is int&, doesn't match int&&